cloud/pkg/synccontroller/objectsync.go

```diff
@@ -68,6 +68,7 @@ func (sctl *SyncController) manageObject(sync *v1alpha1.ObjectSync) {
 			Group:    "",
 			Resource: resource,
 		}, sync.Spec.ObjectName)
+		object.SetUID(types.UID(syncObjUID))
```

Comment:
The change is tricky， it is strange and incomprehensible， we should update the object UID

Reply:
why not do like this?
```
	newObject := &unstructured.Unstructured{}
		newObject.SetNamespace(sync.Namespace)
		newObject.SetName(sync.Spec.ObjectName)
		newObject.SetUID(types.UID(getObjectUID(sync.Name)))
		if msg := buildEdgeControllerMessage(nodeName, sync.Namespace, resourceType, sync.Spec.ObjectName, model.DeleteOperation, newObject); msg != nil {
			beehiveContext.Send(commonconst.DefaultContextSendModuleName, *msg)
		}
```

Reply:
Yes, i think this change is tricky too. If we send delete message and return here, then we are not need to construct NotFound error. Further more, sendEvents first argument is not need.